keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/SecurityGroupHandler.kt

```diff
@@ -283,7 +306,7 @@ class SecurityGroupHandler(
         "vpcId" to cloudDriverCache.networkBy(
           vpcName,
           account,
-          spec.region
+          spec.locations.regions.first()
```

Comment:
Doesn't this need to map to all the regions, not just the first?

Reply:
A `SecurityGroup` spec which may address multiple regions always resolves to a `Map<String, SecurityGroup>` of single region resources. I didn't want to have a combination of `regions: Set<String>` and `region: String` properties in the spec but the assumption is that regions is a single item set regardless of what's in the spec everywhere you see `regions.first()`. Definitely needs some additional tests to validate.